huaweicloud/services/cc/resource_huaweicloud_cc_global_connection_bandwidth.go

```diff
@@ -0,0 +1,405 @@
+	}
+
+	updateChanges := []string{
+		"name",
+		"size",
+		"charge_mode",
+		"sla_level",
+		"binding_service",
+		"spec_code_id",
+		"description",
+	}
+
+	if d.HasChanges(updateChanges...) {
+		err = updateGCB(client, d, cfg.DomainID)
+		if err != nil {
+			return diag.FromErr(err)
+		}
+	}
+
+	if d.HasChanges("tags") {
+		tagErr := updateTags(client, d)
+		if tagErr != nil {
+			return diag.Errorf("error updating tags of global connection bandwidth (%s): %s", d.Id(), tagErr)
+		}
+	}
+
+	return resourceGlobalConnectionBandwidthRead(ctx, d, meta)
+}
+
+func updateTags(client *golangsdk.ServiceClient, d *schema.ResourceData) error {
+	oRaw, nRaw := d.GetChange("tags")
+	oMap := oRaw.(map[string]interface{})
+	nMap := nRaw.(map[string]interface{})
+
+	manageTagsHttpUrl := "v3/gcb/{resource_id}/tags/{action}"
+	manageTagsPath := client.Endpoint + manageTagsHttpUrl
+	manageTagsPath = strings.ReplaceAll(manageTagsPath, "{resource_id}", d.Id())
+	manageTagsOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		OkCodes: []int{
+			204,
+		},
+	}
+
+	// remove old tags
+	if len(oMap) > 0 {
+		manageTagsOpt.JSONBody = map[string]interface{}{
+			"tags": utils.ExpandResourceTags(oMap),
+		}
+		deleteTagsPath := strings.ReplaceAll(manageTagsPath, "{action}", "delete")
+		_, err := client.Request("POST", deleteTagsPath, &manageTagsOpt)
+		if err != nil {
+			return err
+		}
+	}
+
+	// set new tags
+	if len(nMap) > 0 {
+		manageTagsOpt.JSONBody = map[string]interface{}{
+			"tags": utils.ExpandResourceTags(nMap),
+		}
+		createTagsPath := strings.ReplaceAll(manageTagsPath, "{action}", "create")
+		_, err := client.Request("POST", createTagsPath, &manageTagsOpt)
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func updateGCB(client *golangsdk.ServiceClient, d *schema.ResourceData, domainID string) error {
+	updateGCBHttpUrl := "v3/{domain_id}/gcb/gcbandwidths/{id}"
+	updateGCBPath := client.Endpoint + updateGCBHttpUrl
+	updateGCBPath = strings.ReplaceAll(updateGCBPath, "{domain_id}", domainID)
+	updateGCBPath = strings.ReplaceAll(updateGCBPath, "{id}", d.Id())
+	updateGCBOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		JSONBody: map[string]interface{}{
+			"globalconnection_bandwidth": utils.RemoveNil(buildUpdateGCBParams(d)),
+		},
+	}
+
+	_, err := client.Request("PUT", updateGCBPath, &updateGCBOpt)
+	if err != nil {
+		return fmt.Errorf("error updating global connection bandwidth: %s", err)
+	}
+
+	return nil
+}
+
+func buildUpdateGCBParams(d *schema.ResourceData) map[string]interface{} {
+	params := map[string]interface{}{
+		"name":            d.Get("name"),
+		"size":            d.Get("size"),
+		"charge_mode":     d.Get("charge_mode"),
+		"sla_level":       utils.ValueIngoreEmpty(d.Get("sla_level")),
+		"binding_service": utils.ValueIngoreEmpty(d.Get("binding_service")),
+		"spec_code_id":    utils.ValueIngoreEmpty(d.Get("spec_code_id")),
+		"description":     utils.ValueIngoreEmpty(d.Get("description")),
```

Comment:
the description should support clear
```suggestion
		"description":     d.Get("description"),
```